bleak/backends/bluezdbus/manager.py

```diff
@@ -1035,22 +1029,16 @@ def _parse_msg(self, message: Message) -> None:
                                     message_path, new_value
                                 )
 
-    def _run_advertisement_callbacks(
-        self, device_path: str, device: Device1, changed: Iterable[str]
-    ) -> None:
+    def _run_advertisement_callbacks(self, device_path: str, device: Device1) -> None:
```

Comment:
changed has been unused for a while

Reply:
Can you split that out to a separate patch?

Reply:
https://github.com/hbldh/bleak/pull/1633